components/pill-container/private/selected-listbox.jsx

```diff
@@ -94,20 +179,16 @@ const SelectedListBox = (props) =>
 			>
 				{props.selection.map((option, renderIndex) => {
 					const setActiveBasedOnstateFromParent =
-						renderIndex === props.activeOptionIndex &&
-						isEqual(option, props.activeOption);
```

Comment:
What's the reason for removal of this line?

Reply:
@interactivellama it's actually a modification to:
`const setActiveBasedOnStateFromParent = renderIndex === props.activeOptionIndex;`

It's unnecessary to check whether an individual option in a particular iteration deeply equals the activeOption as `props.selection` is an array and therefore consistently ordered, so `renderIndex` is always reliable when compared to `props.activeOptionIndex`. Since `activeOption` and `activeOptionIndex` are always set at the same time by the parent combobox and pill-container components and selected-listbox is a private component, it's impossible for these things to ever get out of sync. It also didn't seem appropriate for the selected-listbox to care about the contents of a particular option while managing focus, which only requires position. Ultimately simplifying that line makes it easier to read and marginally improves performance.